README.md

```diff
@@ -1,2 +1,14 @@
 # sphinx-sharedconf
-Share a conf.py file between several projects
+
+[![CircleCI](https://circleci.com/gh/readthedocs/sphinx-sharedconf/tree/main.svg?style=svg)](https://circleci.com/gh/readthedocs/sphinx-sharedconf/tree/main)
+[![Documentation Status](https://readthedocs.org/projects/sphinx-sharedconf/badge/?version=latest)](https://sphinx-sharedconf.readthedocs.io/en/latest/?badge=latest)
+
+Share a single `conf.py` file between several Sphinx projects.
+useful if you want to import several projects from the same
+repository on [Read the Docs](https://readthedocs.org).
```

Comment:
I like `sphinx-readthedocs-multiproject` :D

Reply:
but it's also kind of long, so maybe sphinx-multiproject is okay?

Reply:
heh yeah, that is quite long. `sphinx-multiproject` seems like a good plan :+1: 

I could see us eventually maybe having a small collection of extensions, and maybe we have a `sphinx-readthedocs` at that point. This seems like it would be a good fit in that umbrella, as you're right that this is probably only really applicable to projects on RTD.